src/Chapter2.hs

```diff
@@ -336,7 +347,8 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList a b xs = if a > b then [] else
```

Comment:
Your implementation is correct when the right bound is lower than the left. But I think you also check on negative bounds here too 👌🏼